main.go

```diff
@@ -65,10 +61,10 @@ func main() {
 	updates := make(chan *registry.ServiceUpdate)
 	done := make(chan struct{})
```

Comment:
This is more commonly called either `ctrl` or `exit`, as this is your "shutdown all running processes" control structure.

Reply:
All I can say here: ok 😄
Looks like now this can be done with `context`, but this is a bit out of my scope of work.